mucgpt-frontend/src/pages/home/Home.module.css

```
@@ -0,0 +1,361 @@
+    margin-top: 72px;
+    color: var(--theme-text-default);
+    text-shadow: 0 2px 10px rgba(57, 91, 82, 0.08);
+    letter-spacing: -0.5px;
+    line-height: 1.2;
+    position: relative;
+    display: block;
+    max-width: 100%;
+    padding: 0 20px;
+    box-sizing: border-box;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)